src/extensions/snapshot/dedent.ts

```diff
@@ -1,15 +1,8 @@
-'use strict'
-
-/**
- * @module extensions/snapshot/dedent
- */
-
 /**
  * Extract spaces length
- * @param {string} text
- * @returns {number} length tab and space before first char
  */
-const getSpacesLength = (text) => {
+const getSpacesLength = (text?: string): number => {
+    if (!text) return 0
     let length = 0
 
     while (length < text.length) {
```

Comment:
```suggestion
let length = 0
    while (length < text.length && (text[length] === ' ' || text[length] === '\t')) {
        length += 1
    }
    return length
```

Reply:
It seems less clear with that condition for the while loop. I prefert to keep the actual code. (We have just 2 lines more)

Reply:
Ok, no worry